### PR TITLE
Update netci.groovy to enforce certificate tests in CI

### PR DESCRIPTION
**Pull request by iamjasonp**
Add switches

/p:SSL_Available=true /p:Root_Certificate_Installed=true
/p:Client_Certificate_installed=true

to CI so tests will always run certificate tests

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -193,14 +193,14 @@ def supportedFullCycleOuterloopPlatforms = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu
             if (osGroupMap[os] == 'Windows_NT') {
                 newJob.with {
                     steps {
-                        batchFile("build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroupMap[os]} /p:WithCategories=OuterLoop /p:ServiceUri=%WcfServiceUri%")
+                        batchFile("build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroupMap[os]} /p:WithCategories=OuterLoop /p:ServiceUri=%WcfServiceUri% /p:SSL_Available=true /p:Root_Certificate_Installed=true /p:Client_Certificate_Installed=true")
```
Comment (roncain) [inline netci.groovy]: Would you mind adding Peer_Certificate_Installed to this list?  If it is set and there is a problem with the peer certificate, the infrastructure tests will tell us what the problem was.

Comment (iamjasonp) [inline netci.groovy]: Added.

**Comment by iamjasonp** [comment]
Fixes #1401

**Comment by roncain** [comment]
LGTM

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
Closing/reopening so I can "test ci please"...

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Pull request merged**